src/python/turicreate/test/test_drawing_classifier.py

```diff
@@ -282,10 +282,28 @@ def test_save_and_load(self):
                     and (new_preds == old_preds).all())
 
     def test_export_coreml(self):
+        import coremltools
+        import platform
         for model in self.models:
             filename = _mkstemp("bingo.mlmodel")[1]
             model.export_coreml(filename)
 
+            # Load the model back from the CoreML model file
+            coreml_model = coremltools.models.MLModel(filename)
+            self.assertDictEqual({
+                'com.github.apple.turicreate.version': _tc.__version__,
+                'com.github.apple.os.platform': platform.platform(),
+                'target': self.target,
+                'feature': self.feature,
+                'type': 'drawing_classifier',
+                'max_iterations': '20',
+                'version': '2',
+                }, dict(coreml_model.user_defined_metadata)
+            )
+            expected_result = 'Drawing classifier created by Turi Create (version %s)' \
```

Comment:
I bet this will fail since we didn't add version info for DC. I will add it.

Reply:
It's added as part of this PR.

Reply:
Ah, need to scroll down more. ;-)